radicle/src/cob/patch.rs

```diff
@@ -376,6 +384,16 @@ impl store::FromHistory for Patch {
                             .into(),
                             op.clock,
                         );
+                        let doc = repo.identity_doc_at(op.identity)?;
```

Comment:
Should the doc be verified somewhere?

Reply:
The doc is verified on fetch, so it shouldn't be necessary to verify again here.